test/unit/walk-manifest.spec.js

```diff
@@ -24,12 +24,20 @@ const customError = function(errors) {
 
 describe('walk-manifest', function() {
   describe('walkPlaylist', function() {
+    /* eslint-disable no-console */
+    beforeEach(function() {
+      this.oldError = console.error;
```

Comment:
We log errors about duplicate segments being downloaded, but its not an issue and we  should prevent it from happening.